system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
```

Comment:
@erikh what is the usecase for these start/stop utilities?

As of now we start them as part of Vagrantfile (with appropriate arguments set). And usually there was never a need to kill these during systemtests as we use cleanup scripts for cleaning up any testcase related state. Do you want something similar?